src/handler-context/Config/__mocks__/index.ts

```diff
@@ -0,0 +1,45 @@
+import { ConfigFiles } from "../types";
+
+// TODO: fill in the rest of these
+const config: ConfigFiles = {
```

Comment:
this will act as the default mock of the config. If you just do a `jest.mock("src/handler-context/Config/__mocks__/index.ts")` then you'll get this.

Reply:
yep absolutely right, dud copy and paste from me there 😅